dockernet/tests/integration_tests.bats

```diff
@@ -199,6 +193,56 @@ setup_file() {
   assert_equal "$stake_diff" "1"
 }
 
+@test "[INTEGRATION-BASIC-$CHAIN_NAME] LSM liquid stake" {
+  if [[ "$CHAIN_NAME" != "LSM" ]]; then
+    skip "Skipping LSM liquid stake for chains without LSM support" 
+  fi
+
+  staker_address_on_host=$($HOST_MAIN_CMD keys show $USER_ACCT -a)
+  staker_address_on_stride=$($STRIDE_MAIN_CMD keys show $USER_ACCT -a)
+  validator_address=$(GET_VAL_ADDR $CHAIN_NAME 1)
+
+  # delegate on the host chain
+  $HOST_MAIN_CMD tx staking delegate $validator_address ${TRANSFER_AMOUNT}${HOST_DENOM} --from $USER_ACCT -y
+  WAIT_FOR_BLOCK $STRIDE_LOGS 2
+
+  # tokenize shares
+  $HOST_MAIN_CMD tx staking tokenize-share $validator_address ${TRANSFER_AMOUNT}${HOST_DENOM} $staker_address_on_host --from $USER_ACCT -y --gas 1000000
+  WAIT_FOR_BLOCK $STRIDE_LOGS 2
+
+  # get the record id from the tokenized share record
+  record_id=$($HOST_MAIN_CMD q staking last-tokenize-share-record-id | awk '{print $2}' | tr -d '"')
+
+  # transfer LSM tokens to stride
+  $HOST_MAIN_CMD tx ibc-transfer transfer transfer $HOST_TRANSFER_CHANNEL \
+    $staker_address_on_stride ${TRANSFER_AMOUNT}${validator_address}/${record_id} --from $USER_ACCT -y
```

Comment:
Store the ${validator_address}/${record_id} in a semantically meaningful name like lsm_token_host_denom

Reply:
done